.github/workflows/pr-function-tests.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
Would checking out the pull request put us at a greater security risk?

Reply:
We would need to authorize it by applying the label. As long as it does not run every time and is approved by us as Maintainers, I don't think it is a security risk.